lib/stdlib/src/erl_lint.erl

```diff
@@ -317,6 +317,10 @@ format_error({unsafe_var,V,{What,Where}}) ->
 format_error({exported_var,V,{What,Where}}) ->
     io_lib:format("variable ~w exported from ~w ~s",
                   [V,What,format_where(Where)]);
+format_error({match_underscore_var, V}) ->
+    io_lib:format("variable ~w is already bound. If you meant to ignore this "
```

Comment:
"meant" should be "mean", to be in agreement with "use".

Also, I'm not sure about "just", but it doesn't roll of my tongue. I'd use "simply" (or perhaps "a single").